### ISSUE TITLE
The result seems not correct

### ISSUE THREAD

**Issue by betachat** [guillaumegenthial/sequence_tagging#61]
I finish train and run evaluate.py, but I got fellowing result:
```bash
input> I love Paris
I love   Paris 
O I-MISC O     
```
but the Accuracy is almost 100%,what's wrong?

sorry for bad English :(

**Comment by betachat** [comment]
I set dropout to 5, and got better results.

**Comment by krishnavinayv1** [comment]
I think the data which is in the folder is very small to train. U need to develop ur own training set and then train and check and it will give nice results hopefully.
